onnx_tf/handlers/backend/sign.py

```diff
@@ -4,12 +4,55 @@
 from onnx_tf.handlers.handler import onnx_op
 from onnx_tf.handlers.handler import tf_func
 from .math_mixin import BasicMathMixin
+from onnx_tf.common import sys_config
+from onnx_tf.common import exception
+from onnx_tf.common import data_type
 
 
 @onnx_op("Sign")
 @tf_func(tf.sign)
 class Sign(BasicMathMixin, BackendHandler):
+  cast_map = {
+    tf.uint8: tf.int32,
+    tf.uint16: tf.int32,
+    tf.uint32: tf.int64,
+    tf.int8: tf.int32,
+    tf.int16: tf.int32,
+    tf.float16: tf.float32
```

Comment:
tf.float16 is a supported dtype for tf.sign, no need to cast it to tf.float32

Reply:
I have addressed this. Thanks for catching it.